### DISCUSSION TITLE
Dislike New Pointer Locking Feature on Navigation

### DISCUSSION THREAD

**Discussion by edpuskas** [mscherotter/galeryst#29] [Ideas]
Greetings. I appreciate the new features being added. But being primarily a Mac user (as well as most of my friends), I am not a fan of the new Pointer Locking feature in the navigation. While I can understand while some may like the feature, it is irritating  with only the trackpad on a Mac. Also using escape to turn off doesn't work well. I usually have to hit escape multiple times and then when I move the pointer comes back. I would recommend putting the cursor back as default and making it an option to use the pointer for those that want it.

**Comment by mscherotter** [comment]
Ed - thank you so much for the feedback.  I implemented the pointer locking based on lots of feedback,
I did take your advice to make it an option in the gallery settings.  I will watch my telemetry to see which option people use more.  Navigating in 3D on a 2D screen is new for some people and common for some.  Finding a good UI that works for both audiences is tricky.

![image](https://user-images.githubusercontent.com/641702/151294788-13343727-636a-4124-ba7a-fdf79f04fb58.png)

**Reply by edpuskas** [reply]
Michael, Thanks for implementing the Lock Pointer as a setting. It works But the little pointer (Circle) still remains in the view of the assets on the wall even though the pointer is turned off. Is it possible to only show the pointer if the Lock Pointer option is chosen. Right now I have it turned off and the cursor is now showing and controlling but the circle/pointer image still shows on the screen.

**Reply by mscherotter** [reply]
I'll fix that this weekend.
Thanks,
Michael

**Reply by mscherotter** [reply]
Fixed - you may need to refresh the page (F5) to update the code the first time you try it out.
Thanks!
Michael

**Comment by edpuskas** [comment]
Thanks very much Michael! Yes I understand it's a good option for many, so having both options is excellent. 
Ed

**Comment by edpuskas** [comment]
Great, thanks for the fast response!
